Function Development/Data Tracking/templates/index.html

```
@@ -470,74 +460,116 @@
+    <div class="stat-card">
+      <div class="stat-icon" style="background:linear-gradient(135deg,#f3e8ff,#e9d5ff);color:#9333ea">
+        <i class="bi bi-align-middle"></i>
+      </div>
+      <div><div class="stat-val" id="s-median">—</div><div class="stat-lbl">中位数 Median</div></div>
+    </div>
+  </div>
+
+  <!-- 图表工具栏 -->
+  <div class="chart-toolbar mb-2" id="chart-toolbar" style="display:none;">
+    <div class="chart-type-btns">
+      <button class="btn-chart-type active" data-type="line">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)